kubernetes/helm/discoveryserver/dev.values.yaml

```diff
@@ -0,0 +1,94 @@
+# Default values for discoveryserver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 3
+
+image:
+  repository: gcr.io/etcd-io-dev/discoveryserver
+  tag: latest
+  pullPolicy: Always
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podSecurityContext:
+  runAsUser: 9001
+  runAsGroup: 9001
+  fsGroup: 9001
+
+securityContext:
+  allowPrivilegeEscalation: false
+
+environment:
+  - name: DISC_ETCD
+    value: "http://etcd-cluster-client.etcd-operator.svc.cluster.local:2379"
```

Comment:
Does the etcd-operator put the etcd cluster in the operator's namespace?  This seems wrong.  Shouldn't the etcd clusters live in a separate namespace from the operator?

Reply:
The default is that the operator is going to manage cluster in the same namespace.
From etcd-operator docs:
_Default etcd operator behavior is to only manage etcd clusters created in the same namespace. It is possible to deploy an etcd operator with special option to manage clusterwide etcd clusters._

Reply:
@eduartua A couple of nitpicks, then. If we want to use a namespaced etcd cluster operator, and we want to keep the etcd cluster within the namespace as the discoveryserver, then we should put the etcd-operator, etcd and discovery server into the namespace, like a namespace called `discoveryserver`. The etcd cluster should be named something descriptive to its purpose, like `discoveryserver-etcd`.

`etcd-cluster-client` is a bad name.

Reply:
I do agree. I've made the changes to have them all running in the same namespace. Etcd cluster name has been changes to `discovery-etcd-cluster`